packages/clay-core/stories/OverlayMask.stories.tsx

```diff
@@ -0,0 +1,144 @@
+/**
+ * SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+const spritemap = require('@clayui/css/lib/images/icons/icons.svg');
+import {ClayCheckbox} from '@clayui/form';
+import ClayLayout from '@clayui/layout';
+import ClayPopover from '@clayui/popover';
+import {storiesOf} from '@storybook/react';
+import React, {useState} from 'react';
+
+import {Provider} from '../src';
+import {OverlayMask} from '../src/overlay-mask';
+
+type Props = {
+	onClick: () => void;
+};
+
+// Use only for demo.
+const Dot = ({onClick}: Props) => (
+	<div
```

Comment:
![image](https://user-images.githubusercontent.com/7663513/162311238-b7046c35-dbbe-4d32-b9ea-103e2fc39d6f.png)

I think this component is just used on story but ppl use to hard copy and paste those examples. Why we don't use ClaySticker with `sticker-outside` here? 🙈

Reply:
I thought about using something else but I was a little afraid of making an impression that people would think we are not following the recommendation, the comment's attempt was to try to prevent people from copying I hope it works, and also the ugliness of it should help 😂